dd-sdk-android/src/test/kotlin/com/datadog/android/log/LoggerTest.kt

```diff
@@ -366,7 +370,7 @@ internal class LoggerTest {
                 Log.INFO,
                 fakeMessage,
                 null,
-                mapOf(key to value),
+                emptyMap(),
```

Comment:
You're breaking the way logs work here. When they look the logs attributes online, instead of seeing `my_attribute: null`, they won't see `my_attribute` at all :/

Reply:
This collection does not accept "null" as values and it will throw an NPE, what if instead of NULL we add a "null" as String ?

Reply:
That won't work, since `"null"` != `null`
What you can do is create a `val NULL = Object()`, and pass it instead, and when serializing convert that to JsonNull.